src/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title><%= process.env.MY_APP %></title>
+        <style>
+            html, body {
+                position: relative;
+                margin: 0;
+                padding: 0;
+            }
+
+            * {
+                box-sizing: border-box;
+            }
+
+            #app {
+                font-family: 'Montserrat', sans-serif;
+            }
+        </style>
+    </head>
+
+    <body>
+        <noscript>
+            <%= process.env.MY_APP %> needs JS.
+        </noscript>
+        <div id="app"></div>
+
+        <link href="https://fonts.googleapis.com/css?family=Oxygen+Mono|Montserrat:300,400,600" rel="stylesheet">
+
+        <!-- Start of anti-click-jacking -->
+        <script type="text/javascript">
+            if (self === top) {
+                var antiClickjack = document.getElementById("anti-click-jacking");
+                antiClickjack.parentNode.removeChild(antiClickjack);
+            } else {
+                top.location = self.location;
+            }
+        </script>
+        <!-- End of anti-click-jacking -->
+
+        <!-- Start of service-worker -->
```

Comment:
Remove these comments

Reply:
These are proper comments.
When we request to remove comments, it's generally commented out code.